components/RadioButtons.js

```diff
@@ -0,0 +1,19 @@
+const RadioButtons = ({ name }) => {
+  const createRadioButton = props.options.map((option) => {
+    return (
+      <div>
+        <label>{option}</label>
+        <input type="radio" value={option} key={option} name={props.name} />
+      </div>
+    );
+  });
+
+  return (
+    <div>
+      <label>{name}</label>
+      <div onChange={handleChange}>{createRadioButton}</div>
+    </div>
+  );
+};
+
```

Comment:
@pasternakewa please analyze this component and try to divide it more precisely.